Resources/public/js/models/ez-contentmodel.js

```diff
@@ -240,8 +244,13 @@ YUI.add('ez-contentmodel', function (Y) {
                             locations.push(location);
                         });
 
-                    location = new Y.eZ.Location({id: loc._href});
-                    location.load(options, end);
+                    if(currentLocation && currentLocation.get('id') === loc._href) {
+                        location = currentLocation;
+                        end();
```

Comment:
since that's absolutely not asynchronous, in this case we should not create the `end` function and then not call it :)